bin/compile

```diff
@@ -384,6 +384,11 @@ case $TOOL in
     ;;
 esac
 
+if [ ! -z ${name+x} ];
```

Comment:
In the case of the gb package manager being used, it seems that `name` would not be set.